recipes/exifread/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set pypi_name = "ExifRead" %}
+{% set name = pypi_name|lower %}
```

Comment:
Just tried the URL below and it worked for me. 😕 

https://pypi.io/packages/source/E/ExifRead/ExifRead-2.1.2.tar.gz

Though have had annoyances with these URLs before. Hence the way the [example]( https://github.com/conda-forge/staged-recipes/blob/1abe1dc23337e07188117e6db77014321f8677a0/recipes/example/meta.yaml ) is now. Maybe worth giving that a try.